bin/run-tests.sh

```diff
@@ -18,7 +18,7 @@ for test_dir in tests/*; do
     test_dir_name=$(basename "${test_dir}")
     test_dir_path=$(realpath "${test_dir}")
 
-    bin/run.sh "${test_dir_name}" "${test_dir_path}" "${test_dir_path}"
+    bash -x bin/run.sh "${test_dir_name}" "${test_dir_path}" "${test_dir_path}"
```

Comment:
This verbose debugging output can be removed once we go live. I think not being able to deploy the image was causing me problems before.

This debugging only clutters up the output for the GitHub workflow. Does not affect student solutions